workbench/src/renderer/components/PluginModal/index.jsx

```diff
@@ -63,34 +63,29 @@ export default function PluginModal(props) {
     <Modal.Body>
       <Form>
         <Form.Group className="mb-3">
-          <Form.Label htmlFor="url">Add a plugin</Form.Label>
+          <Form.Label htmlFor="url">{t('Add a plugin')}</Form.Label>
           <Form.Control
             id="url"
-            name="url"
-            type="text"
```

Comment:
Would you mind adding `type="text"` back to this form control? ([When `type` is omitted, it does default to `"text"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#type), but it's a best practice to always include a `type` attribute on an `<input>`.)